### PR TITLE
chore: add missing linting rules from Podman Desktop

### PR DESCRIPTION
**Pull request by benoitf**
fixes issues reported by using the same rules

it also fixes https://github.com/projectatomic/studio-extension/issues/97

avoid to see 
image

### CODE REVIEW

File: packages/backend/src/managers/catalogManager.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class CatalogManager {
     try {
       this.watchCatalogFile(catalogPath); // do not await, we want to do this async
     } catch (err: unknown) {
-      console.error("unable to watch catalog file, changes to the catalog file won't be reflected to the UI", err);
+      console.error(`unable to watch catalog file, changes to the catalog file won't be reflected to the UI`, err);
```
Comment (axel7083) [inline packages/backend/src/managers/catalogManager.ts:69]: Not a template ?

File: packages/backend/src/models/AIConfig.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ export interface AIConfig {
 }
 
 export function isString(value: unknown): value is string {
+  // eslint-disable-next-line no-null/no-null
   return (value !== undefined && value !== null && typeof value === 'string') || value instanceof String;
```
Comment (axel7083) [inline packages/backend/src/models/AIConfig.ts:36]:
```suggestion
  return (!!value && typeof value === 'string') || value instanceof String;
```

File: packages/shared/src/messages/MessageProxy.ts [hunk 3]
```diff
@@ -43,6 +43,7 @@ type UnaryRPC = (...args: unknown[]) => Promise<unknown>;
 
 export function isMessageRequest(content: unknown): content is IMessageRequest {
   return (
+    // eslint-disable-next-line no-null/no-null
     content !== undefined && content !== null && typeof content === 'object' && 'id' in content && 'channel' in content
```
Comment (axel7083) [inline packages/shared/src/messages/MessageProxy.ts:47]:
```suggestion
    !!content && typeof content === 'object' && 'id' in content && 'channel' in content
```

File: packages/shared/src/messages/MessageProxy.ts [hunk 4]
```diff
@@ -210,6 +211,7 @@ export class RpcBrowser {
   isSubscribedMessage(content: any): content is ISubscribedMessage {
     return (
       content !== undefined &&
+      // eslint-disable-next-line no-null/no-null
       content !== null &&
```
Comment (axel7083) [inline packages/shared/src/messages/MessageProxy.ts:215]:
```suggestion
      !!content &&
```

**Review by axel7083 — APPROVED** [review]
LGTM

**Review by lstocchi — APPROVED** [review]
I think we can remove the null checks, but ok for a follow-up PR. LGTM

File: packages/backend/src/managers/catalogManager.ts (hunk 1 above)
Comment (benoitf) [reply packages/backend/src/managers/catalogManager.ts:69]: else we need to replace won't by will not

File: packages/backend/src/models/AIConfig.ts [hunk 5]
```diff
@@ -32,7 +32,8 @@ export interface AIConfig {
 }
 
 export function isString(value: unknown): value is string {
-  return (value !== undefined && value !== null && typeof value === 'string') || value instanceof String;
+  // eslint-disable-next-line no-null/no-null
```
Comment (benoitf) [inline packages/backend/src/models/AIConfig.ts:35]:
```suggestion
```

File: packages/shared/src/messages/MessageProxy.ts [hunk 6]
```diff
@@ -43,7 +43,8 @@ type UnaryRPC = (...args: unknown[]) => Promise<unknown>;
 
 export function isMessageRequest(content: unknown): content is IMessageRequest {
   return (
-    content !== undefined && content !== null && typeof content === 'object' && 'id' in content && 'channel' in content
+    // eslint-disable-next-line no-null/no-null
```
Comment (benoitf) [inline packages/shared/src/messages/MessageProxy.ts:46]:
```suggestion
```

**Pull request merged**